consensus/poet/core/tests/test_consensus/test_wait_certificate.py

```diff
@@ -164,49 +173,60 @@ def test_create_after_wait_timer_timed_out(self):
         with self.assertRaises(ValueError):
             WaitCertificate.create_wait_certificate(
                 poet_enclave_module=self.poet_enclave_module,
+                sealed_signup_data=signup_info.sealed_signup_data,
                 wait_timer=wt,
                 block_hash="Reader's Digest")
 
-    def test_create_with_wrong_wait_timer(self):
+    def test_create_with_wrong_signup_data(self):
```

Comment:
I would have thought we would want the simulator to move closer to mirroring the functionality of the enclave not further away. Regardless of whether an MC simulation could survive a reboot, the interesting runtime features of being able to invalidate wait timers is a trival implementation and llows for those expect features to be tested on both implementations. 

These tests should be run on both the enclave and the simulator.